deployment/keystone/deploy.go

```diff
@@ -425,30 +425,19 @@ type RegisteredCapability struct {
 	ID [32]byte
 }
 
-func FromCapabilitiesRegistryCapability(cap *kcr.CapabilitiesRegistryCapability, e deployment.Environment, registryChainSelector uint64) (*RegisteredCapability, error) {
```

Comment:
No other consumers?

Reply:
thanks. yes there are. i didn't mean to delete this